index.html

```diff
@@ -26,7 +26,7 @@ <h1 id="logo">Aziz Mejri</h1>
         </ul>
       </nav>
```

Comment:
- [ ] It would be nice to have the close button position and CSS properties follow the design from the [chosen template in Figma](https://www.figma.com/file/t3EJUCAEViw3QasuJLPLVT/Microverse-Student-Potfolio-Templates-Main?node-id=34%3A1667). This is one of the requirements but most of all, it will improve user experience.

### Currently you have:
![your-nav](https://user-images.githubusercontent.com/26796167/147244379-065d1718-3193-4238-b31d-6bb7ed1214bf.PNG)

### Instead of:
![nav](https://user-images.githubusercontent.com/26796167/147244414-1234eadc-8d30-4595-9d80-75bc2d1d6eab.PNG)